prow/statusreconciler/status.go

```diff
@@ -103,12 +109,12 @@ func (s *statusController) loadState() (storedState, error) {
 	entry := s.logger.WithField("path", s.statusURI)
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
-	reader, err := s.opener.Reader(ctx, s.statusURI)
```

Comment:
The primary thing I want is to keep the interface like this, aka we shoudn't be passing bucket options to an abstract Reader interface.

Reply:
Done > https://github.com/kubernetes/test-infra/pull/16317/commits/b767f4531f54a8e65753d4da3b0a38f1219cd4f9